asistente_ladm_col/tests/resources/sql/empty_no_interlis_no_ladm.sql

```diff
@@ -0,0 +1 @@
+IF NOT EXISTS (SELECT  schema_name FROM information_schema.schemata WHERE schema_name = 'empty_no_interlis_no_ladm')EXEC sp_executesql N'CREATE SCHEMA empty_no_interlis_no_ladm';
```

Comment:
Please make a clear distinction between PG and MSSQL in `resources/sql/`, otherwise it will get confused.